libs/hdf-converters/package.json

```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "@mitre/splunk-sdk-no-env": "^1.10.0",
     "@types/csv2json": "^1.4.2",
+    "@types/mustache": "^4.1.2",
     "@types/papaparse": "^5.3.2",
     "@types/xml2js": "^0.4.9",
     "aws-sdk": "^2.1046.0",
```

Comment:
Does the package.json need to include `'@aws-sdk/client-config-service'`? After a clean npm install within hdf-converters, the test failed because it couldn't find this package. It is listed in the `package.json` file for Heimdall, but should you need to do `npm install` from the top layer to be able to run the tests in `hdf-converters`?

Reply:
It shouldn't require this, the AWS config mapper uses the SDK as exported by aws-sdk: https://github.com/mitre/heimdall2/blob/master/libs/hdf-converters/src/aws-config-mapper.ts#L7